python/tests/compat/conftest.py

```diff
@@ -0,0 +1,142 @@
+import pytest
+import subprocess
+import os
+import venv
+import tempfile
+
+class ErrorInVenv(Exception):
+    """To signal errors occuring from within the venv"""
+
+def run_shell_commands(commands_list, cwd=None):
+    command = " && ".join(commands_list)
+    print("Executing command: ", command)
+    result = subprocess.run(command, cwd=cwd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    if result.returncode != 0 or True: # Fix
```

Comment:
This case needs to be sorted out
- The `or True`
- Silent failure

Reply:
Sorry, fixed.